### PR TITLE
Pass access token on game pass submit

### PR DESCRIPTION
**Pull request by maexled**
Part of Gamify-IT/issues#286

### CODE REVIEW

**Review by marlau97 — APPROVED** [review]

File: src/main/java/de/unistuttgart/finitequizbackend/clients/ResultClient.java [hunk 1]
```diff
@@ -13,9 +14,9 @@ public interface ResultClient {
     /**
      * Submits the resultDTO to the Overworld-Backend
      *
+     * @param accessToken the users access token
      * @param resultDTO the player submitted result, trimmed down to the data needed for the overworld
      */
     @PostMapping("/submit-game-pass")
-    @Headers("Content-Type: application/json")
```
Comment (madebyTimo) [inline src/main/java/de/unistuttgart/finitequizbackend/clients/ResultClient.java:19]: Why did you remove the Content-Type?

I saw it is also not preset in the chickenshock-backend but I think it is better to add it in both instead of remove it

Comment (delvh) [reply src/main/java/de/unistuttgart/finitequizbackend/clients/ResultClient.java:19]: My guess: The FeignClient adds it by default when sending an object. But I'm only guessing here.

Comment (maexled) [reply src/main/java/de/unistuttgart/finitequizbackend/clients/ResultClient.java:19]: Just copy pasted from crosswordpuzzle backend. I think it is not really needed.

**Review by LeviUniAcc — APPROVED** [review]

**Comment by delvh** [comment]
Can this PR be merged?

**Comment by maexled** [comment]
> Can this PR be merged?

Yes
